### ISSUE TITLE
Gamebanned..

### ISSUE THREAD

**Issue by BuBiiii22** [ValveSoftware/Dota2-Gameplay#7126]
### Description

Hi, 
295 days ago I got a ban on dot 2, but for no reason at all, I've dealt with it before, but no one answered me and no one dealt with it.. I don't know how to prove I got it for no reason. 1) it looks bad on the profile, 2) because of the ban that I can't see them playing professionally csgo, 3) because of the fact that it's a gameban so I can't go to the side otherwise I'd get a ban there, because of the fact that I have a gameban there. I've been playing dot on my apprenticeship, I've got something like 5000h in there, but I haven't turned the game on in four years. POtom I added the phone number to this account and got instant gameban. I'm begging for this ban to be lifted. I'm not aware of what I've done.  Really from no reason, and i rlly want to be pro player, but i dont want to create a new acc bcs one ban what i dont deserve..
thanks you a lot

ACC:
 https://steamcommunity.com/id/BuBiiii22

### Screenshots

![image](https://user-images.githubusercontent.com/126642122/222044414-607d669c-129b-45d8-804e-63fbd0d10348.png)

**Comment by Usera2** [comment]
It's a bug report and not ban appeal.
Since it's a vac ban contact both steam and game dev

**Comment by sare167** [comment]
Obviously, your article is adequate, but I thought it would be much more effective to include professional photos and videos. Please visit and share your thoughts on [cut mover](https://cutmovergame.com), which features articles and images on these topics.
